_posts/2000-01-10-how.md

```diff
@@ -12,7 +12,9 @@ With [Gradle](http://gradle.org) one can do:
 
 {% highlight Groovy %}
 repositories { jcenter() }
-dependencies { testCompile "org.mockito:mockito-core:2.+" }
+dependencies { testImplementation "org.mockito:mockito-core:2.+" }
+// If using Java 8
+dependencies { testImplementation "org.mockito:mockito-core:3.+" }
```

Comment:
Let's simplify this and replace just the 2 with a 3 now that I think of it.

Reply:
Done 👍